app/src/test/java/org/mozilla/fenix/FenixApplicationTest.kt

```diff
@@ -140,7 +140,7 @@ class FenixApplicationTest {
         every { settings.showTopSitesFeature } returns true
         every { settings.historyMetadataUIFeature } returns true
         every { settings.showPocketRecommendationsFeature } returns true
-        every { settings.showPocketRecommendationsFeature } returns true
+        every { settings.showContileFeature } returns true
```

Comment:
I noticed that the `every` line for `showPocketRecommendationsFeature` was duplicated, so just replaced the dupe with `showContileFeature`

Reply:
Can you add tests to check that we have the correct value for all CustomizeHome telemetry? Thanks

Reply:
The test is failing. After doing some research, it looks like the tests existed previously but were deleted here
https://github.com/mozilla-mobile/fenix/pull/21411/files#r719908594

I'm going to back out the latest test additions if that sounds good to you

Reply:
I see that's unfortunate. Can you make sure to validate through manual testing then? https://github.com/mozilla-mobile/fenix/wiki/Test-telemetry-pings

Reply:
@Amejia481 I was wondering if we had any luck with mocking `experiments` and calling `onUpdatesApplied`?

Reply:
![image](https://user-images.githubusercontent.com/87384386/160466391-1ecb053b-e682-4f2c-9ad2-34074b167705.png)